test/button/button-subtle-perceptual.js

```diff
@@ -0,0 +1,67 @@
+const puppeteer = require('puppeteer');
+const visualDiff = require('visual-diff');
+
+visualDiff.run((ctx) => {
```

Comment:
This sets up some mocha context and place to store the screenshots.  Basically trying to keep the consumer's test file as simple as possible.